src/darwin/Framework/CHIP/MTRDeviceController.mm

```diff
@@ -234,6 +239,8 @@ - (BOOL)startup:(MTRDeviceControllerStartupParamsInternal *)startupParams
 
         _operationalCredentialsDelegate->SetDeviceCommissioner(_cppCommissioner);
 
+        _partialDACVerifier = new chip::Credentials::PartialDACVerifier();
```

Comment:
Should be checking for allocation failure, no?

Reply:
Moved allocation to initWithFactory and checking for allocation failure there.